depth_image_proc/src/disparity.cpp

```diff
@@ -116,7 +116,8 @@ DisparityNode::DisparityNode(const rclcpp::NodeOptions & options)
         std::string topic = node_base->resolve_topic_or_service_name("left/image_rect", false);
         image_transport::TransportHints hints(this);
         sub_depth_image_.subscribe(this, topic, hints.getTransport());
-        sub_info_.subscribe(this, "right/camera_info");
+        sub_info_.subscribe(this, "right/camera_info",
+          rclcpp::QoS(rclcpp::QoSInitialization::from_rmw(rmw_qos_profile_default)));
```

Comment:
I think we can simplify this to:

```suggestion
        sub_info_.subscribe(this, "right/camera_info", rclcpp::QoS(10));
```

(same goes for all of the other instances below)